packages/legend-extension-dsl-serializer/src/components/studio/BindingElementEditor.tsx

```diff
@@ -0,0 +1,450 @@
+              {modelInclude ? `Model Includes` : `Model Excludes`}
+            </div>
+          </div>
+        </div>
+        <div
+          ref={dropElementRef}
+          className="binding-model-editor__panel__content dnd__overlay__container"
+        >
+          <div
+            className={clsx({ dnd__overlay: isElementDragOver && !isReadOnly })}
+          />
+          <div className="binding-model-editor__panel__content__form">
+            <div className="binding-model-editor__panel__content__form__section">
+              <div className="binding-model-editor__panel__content__form__section__header__prompt">
+                {modelInclude
+                  ? `Specifies the list of models included`
+                  : `Specifies the list of models excluded`}
+              </div>
+              <div className="binding-model-editor__panel__content__form__section__list">
+                {modelInclude &&
+                  modelUnitState.packageableElementIncludes.map(
+                    (elementRef) => (
+                      <ElementEditor
+                        key={elementRef.value.uuid}
+                        modelUnitState={modelUnitState}
+                        elementRef={elementRef}
+                        modelInclude={modelInclude}
+                        isReadOnly={isReadOnly}
+                      />
+                    ),
+                  )}
+                {!modelInclude &&
+                  modelUnitState.packageableElementExcludes.map(
+                    (elementRef) => (
+                      <ElementEditor
+                        key={elementRef.value.uuid}
+                        modelUnitState={modelUnitState}
+                        elementRef={elementRef}
+                        modelInclude={modelInclude}
+                        isReadOnly={isReadOnly}
+                      />
+                    ),
+                  )}
+
+                <div className="binding-model-editor__panel__content__form__section__list__new-item__add">
+                  <button
+                    className="binding-model-editor__panel__content__form__section__list__new-item__add-btn btn btn--dark"
+                    disabled={!allowAddingElement}
+                    tabIndex={-1}
+                    onClick={addElement}
+                    title="Add Element"
+                  >
+                    Add Value
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  },
+);
+
+const BindingGeneralEditor = observer(
+  (props: { editorState: BindingEditorState; isReadOnly: boolean }) => {
+    const { editorState, isReadOnly } = props;
+    const editorStore = useEditorStore();
+    const binding = editorState.binding;
+    const schemaSets = Array.from(
+      editorStore.graphManagerState.graph.allOwnElements,
+    ).filter(
+      (type: PackageableElement): type is SchemaSet =>
```

Comment:
petty me, but let's treat variable names like children's names. Let's rename this of `element` instead of `type`.